Source/Core/Core/IOS/USB/Bluetooth/WiimoteDevice.h

```diff
@@ -53,13 +53,14 @@ class WiimoteDevice
   const u8* GetLinkKey() const { return m_LinkKey; }
 
 private:
-  enum ConnectionState
+  enum class ConnectionState
   {
-    CONN_INACTIVE = -1,
-    CONN_READY,
-    CONN_LINKING,
-    CONN_COMPLETE
+    Inactive = -1,
```

Comment:
I honestly don't know. Can't change it without breaking save-states though, since the instance variable for that enum is part of the state data that gets pushed out.